pkg/virt-handler/vm.go

```diff
@@ -177,8 +233,14 @@ func MapPersistentVolumes(vm *v1.VM, restClient cache.Getter, namespace string)
 				newDisk.Source.Protocol = "iscsi"
 
 				hostPort := strings.Split(pv.Spec.ISCSI.TargetPortal, ":")
+				ipAddrs, err := net.LookupIP(hostPort[0])
```

Comment:
What were you thinking of for a better name to 'hostPort' - that accurately reflects what it contains, doesn't it ?